tls/src/lib.rs

```diff
@@ -142,6 +150,10 @@ impl<T> Stream<T> {
     pub fn peer_platform_id(&self) -> &PlatformId {
         &self.platform_id
     }
+
+    pub fn appraisal_status(&self) -> bool {
```

Comment:
nit: Maybe call this `appraisal_success` to be more clear that `true` is success and `false` is not.